spec/integration/resource/api200/volume_attachment/update_spec.rb

```diff
@@ -3,22 +3,10 @@
 RSpec.describe OneviewSDK::VolumeAttachment, integration: true, type: UPDATE do
   include_context 'integration context'
 
-  describe '#get_extra_unmanaged_volumes' do
```

Comment:
Why did you move all the*non-create* actions to the create_spec? It would be really good if they were here in *update_spec*.